### PR TITLE
More typings

### PR DESCRIPTION
**Pull request by miherlosev**
@LavrovArtem

### CODE REVIEW

File: src/client/sandbox/upload/index.ts [hunk 1]
```diff
@@ -100,7 +101,7 @@ export default class UploadSandbox extends SandboxBase {
         return value;
     }
 
-    doUpload (input, filePaths) {
+    doUpload (input: HTMLInputElement, filePaths) {
```
Comment (LavrovArtem) [inline src/client/sandbox/upload/index.ts]: `filePaths: string | Array<string>`

File: src/client/utils/dom.ts [hunk 2]
```diff
@@ -133,15 +133,15 @@ export function getFrameElement (win) {
     }
 }
 
-export function getMapContainer (el) {
+export function getMapContainer (el: HTMLMapElement) {
```
Comment (LavrovArtem) [inline src/client/utils/dom.ts]:
```diff
- el: HTMLMapElement
+ el: HTMLElement
```

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@LavrovArtem FPR

**Pull request merged**